### PR TITLE
PPL-19664: PoC with riak-python-client on Python 3.9

### PR DESCRIPTION
**Pull request by ihor-zhakun**
https://peopleai.atlassian.net/browse/PPL-19664

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -52,8 +56,8 @@ The easiest way to install aioriak is by using the package on PyPi::
 Requirements
 ------------
 
-- Python >= 3.5
-- riak>=2.1.3
+- Python >= 3.9
+- riak>=2.7.0
```
Comment (le-r-on) [inline README.rst]: We should probably mention v3.0.0 from https://github.com/people-ai/riak-python-client/pull/1/files, right?

File: aioriak/client.py [hunk 2]
```diff
@@ -376,17 +375,32 @@ async def get(self, robj):
 
         return await self._transport.get(robj)
 
-    async def put(self, robj, return_body):
+    async def put(self, robj, w=None, dw=None, pw=None, return_body=None,
```
Comment (le-r-on) [inline aioriak/client.py:378]: What are these changes?  Where do they come from?

File: setup.py [hunk 3]
```diff
@@ -16,9 +16,16 @@ def read(*parts):
...
+def req_file(filename):
+    with open(filename) as f:
+        content = f.readlines()
+    return [x.strip() for x in content]
+
+
 setup(
-    name='aioriak',
-    version='0.1.7',
+    name='ppl_aioriak',
```
Comment (le-r-on) [inline setup.py:27]: We should probably revert this back to `aioriak`?

File: user.conf [hunk 4]
```diff
@@ -0,0 +1,2 @@
+search = on
```
Comment (le-r-on) [inline user.conf:1]: Do we need this file in master?

**Review by le-r-on — COMMENTED** [review]
Let's also explain what we did and why in the PR/commit description.

File: setup.py (hunk 3 above)
Comment (ihor-zhakun) [reply setup.py:27]: There is already 'ppl-aioriak' 1.0.0 at codeartifact, so there I don't see a problem to have 2.0.0

File: aioriak/client.py (hunk 2 above)
Comment (ihor-zhakun) [reply aioriak/client.py:378]: just changes from original repo by Belousow Makc

File: user.conf (hunk 4 above)
Comment (ihor-zhakun) [reply user.conf:1]: This file is using only for integration-tests (by docker-compose file)

File: README.rst (hunk 1 above)
Comment (ihor-zhakun) [reply README.rst]: fixed

**Comment by le-r-on** [comment]
Since in this PR you are mixing together lots of your own WIP commits and separate commits from the upstreams - could you please squash them properly (so your Python 3.9 / Docker changes are in a single commit)?

**Review by le-r-on — APPROVED** [review]

**Pull request merged**